frontend/public/components/nav.jsx

```diff
@@ -376,20 +371,17 @@ export class Nav extends React.Component {
             }
             <HrefLink href="/overview" name="Status" activePath="/overview/" onClick={this.close} required={FLAGS.OPENSHIFT} />
             <HrefLink href="/status" name="Status" activePath="/status/" onClick={this.close} disallowed={FLAGS.OPENSHIFT} />
-            <HrefLink href="/catalog" name="Catalog" activePath="/catalog/" onClick={this.close} />
             <HrefLink href="/search" name="Search" onClick={this.close} startsWith={searchStartsWith} />
             <ResourceNSLink resource="events" name="Events" onClick={this.close} />
           </NavSection>
-
-          <NavSection required={FLAGS.OPERATOR_LIFECYCLE_MANAGER} text="Operators" img={operatorImg} activeImg={operatorActiveImg} >
-            {
-              // <HrefLink required={FLAGS.KUBERNETES_MARKETPLACE} href="/marketplace" name="Kubernetes Marketplace" activePath="/marketplace/" onClick={this.close} />
-            }
-            <ResourceNSLink model={ClusterServiceVersionModel} resource={ClusterServiceVersionModel.plural} name="Cluster Service Versions" onClick={this.close} />
+          <NavSection text="Catalog" icon="pficon pficon-catalog">
+            <HrefLink href="/catalog" name="Developer Catalog" activePath="/catalog/" onClick={this.close} />
+            <HrefLink href="/provisionedservices" name="Provisioned Services" activePath="/provisionedservices/" onClick={this.close} />
+            <HrefLink href="/operators" name="Operators" activePath="/operators/" onClick={this.close} />
             <Sep />
-            <ResourceNSLink model={PackageManifestModel} resource={PackageManifestModel.plural} name="Package Manifests" onClick={this.close} />
-            <ResourceNSLink model={SubscriptionModel} resource={SubscriptionModel.plural} name="Subscriptions" onClick={this.close} />
-            <ResourceNSLink model={InstallPlanModel} resource={InstallPlanModel.plural} name="Install Plans" onClick={this.close} />
+            <HrefLink required={FLAGS.KUBERNETES_MARKETPLACE} href="/marketplace" name="Marketplace" activePath="/marketplace/" onClick={this.close} />
```

Comment:
Can we hold off on re-enabling the marketplace nav item until PR #875 goes through?

Reply:
done